pydisdrometer/plot/plot.py

```diff
@@ -32,22 +34,39 @@ def plot_dsd(dsd, range=None, log_scale=True, tighten=True):
     fig_handle: Figure Handle
 
     '''
+    ax =  parse_ax(ax)
+    fig = parse_ax(fig)
```

Comment:
I think you mean parse_fig here? Also if we set it to that, we get 2 plots created.